### PR TITLE
Couple of small fixes

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ documentation](https://libvirt.org/formatdomain.html#elementsCPU).
      aytests-tests):
 
         $ zypper in ruby2.1-rubygem-aytests
-        $ git clone https://github.com/yast/aytests-tests
+        $ git clone https://github.com/yast/autoyast-integration-test.git
```
Comment (imobach) [inline README.md]: This line refers to the tests themselves so the correct repository is https://github.com/yast/aytests-tests.

File: lib/aytests/virtualbox_vm.rb [hunk 2]
```diff
@@ -53,7 +55,8 @@ def mac
     #
     # It relies on `VBoxManage` to update the definition.
     def save
-      Cheetah.run(["VBoxManage", "modifyvm", name, "--macaddress1", @mac] + boot_order_to_options)
+      # VBoxManage requires MAC addresses without ":". It reports invalid MAC otherwise
+      Cheetah.run(["VBoxManage", "modifyvm", name, "--macaddress1", mac.delete(":")] + boot_order_to_options)
```
Comment (imobach) [inline lib/aytests/virtualbox_vm.rb]: Thanks for noticing that!

File: README.md (hunk 1 above)
Comment (mchf) [inline README.md]: ah ok. I'm not good in reading manuals ;-)

**Comment by imobach** [comment]
LGTM. Thanks!

**Comment by mchf** [comment]
Thanks for review (btw no version and change log update needed? ;-)

**Pull request merged**